test/suites/auth.sh

```diff
@@ -1145,6 +1152,65 @@ auth_project_features() {
   lxc project delete blah
 }
 
+auth_ovn() {
+  remote="${1}"
+
+  if [ -z "${LXD_OVN_NB_CONNECTION:-}" ]; then
+    echo "OVN northbound connection not set. Skipping OVN tests..."
+    return
+  fi
+
+  lxc config set network.ovn.northbound_connection "${LXD_OVN_NB_CONNECTION}"
+
+  # If the connection uses SSL we have more required environment variables.
+  # Set the client cert, key, and CA cert.
+  if [[ "${LXD_OVN_NB_CONNECTION}" =~ ^ssl: ]]; then
+    lxc config set network.ovn.client_cert="$(< "${LXD_OVN_NB_CLIENT_CRT_FILE}")"
+    lxc config set network.ovn.client_key="$(< "${LXD_OVN_NB_CLIENT_KEY_FILE}")"
+    lxc config set network.ovn.ca_cert="$(< "${LXD_OVN_NB_CA_CRT_FILE}")"
+  fi
+
+  uplink_network="uplink$$"
+
+  echo "Create a dummy physical network for use as an uplink."
+  ip link add dummy0 type dummy
+  lxc network create "${uplink_network}" --type=physical parent=dummy0
+
+  echo "Set OVN ranges."
+  lxc network set "${uplink_network}" ipv4.ovn.ranges=192.0.2.100-192.0.2.254
+  lxc network set "${uplink_network}" ipv6.ovn.ranges=2001:db8:1:2::100-2001:db8:1:2::254
+
+  echo "Set IP routes that include OVN ranges."
+  lxc network set "${uplink_network}" ipv4.routes=192.0.2.0/24
+  lxc network set "${uplink_network}" ipv6.routes=2001:db8:1:2::/64
+
+  echo "Create a project and grant the fine-grained identity operator access."
+  lxc project create foo -c features.networks=true
+  lxc auth group permission add test-group project foo operator
+  lxc auth group permission add test-group network "${uplink_network}" can_view project=default
+
+  echo "Create an OVN network as the fine-grained identity and check access."
+  lxc network create "${remote}:my-network" --type ovn --project foo network="${uplink_network}"
+  [ "$(lxc network list -f csv "${remote}:" --project foo | wc -l)" = 1 ]
+  [ "$(lxc network list -f csv "${remote}:" --all-projects | wc -l)" = 2 ] # ovn network + uplink
+
+  echo "Delete the OVN network as the fine-grained identity and check access."
+  lxc network delete "${remote}:my-network" --project foo
+  [ "$(lxc network list -f csv "${remote}:" --project foo)" = "" ]
+  [ "$(lxc network list -f csv "${remote}:" --all-projects | wc -l)" = 1 ] # uplink only
+
+  # Clean up
+  lxc network delete "${uplink_network}"
+  ip link delete dummy0
+  lxc project delete foo
+  lxc config unset network.ovn.northbound_connection
+  if [[ "${LXD_OVN_NB_CONNECTION}" =~ ^ssl: ]]; then
+    lxc config unset network.ovn.client_cert
+    lxc config unset network.ovn.client_key
+    lxc config unset network.ovn.ca_cert
```

Comment:
Heh, apparently we cannot batch `lxc config unset`. To do those all in one go we need:

```suggestion
    lxc config set network.ovn.client_cert="" network.ovn.client_key="" network.ovn.ca_cert=""
```
I'll fill a bug :)